packages/scaffold/src/views/w3m-convert-select-network-view/index.ts

```diff
@@ -0,0 +1,88 @@
+import { customElement } from '@web3modal/ui'
```

Comment:
This a component from the hackweek and we currently don't need/use this. We can remove